### DISCUSSION TITLE
Home Weather Station (multi sensors board)

### DISCUSSION THREAD

**Discussion by TrDA-hab** [arendst/Tasmota#10704] [Show and tell]
## My practice:
![](https://raw.githubusercontent.com/TrDA-hab/Projects/master/HomeWeatherStation/Multi-sensor-1.jpg)
![](https://raw.githubusercontent.com/TrDA-hab/Projects/master/HomeWeatherStation/WHM-4.jpg)
![](https://raw.githubusercontent.com/TrDA-hab/Projects/master/HomeWeatherStation/WHM-5.jpg)
![](https://raw.githubusercontent.com/TrDA-hab/Projects/master/HomeWeatherStation/Multi-sensor-4.jpg)
![](https://raw.githubusercontent.com/TrDA-hab/Projects/master/HomeWeatherStation/Untitled%20v38-1.jpg)
![](https://raw.githubusercontent.com/TrDA-hab/Projects/master/HomeWeatherStation/20200529_002351.jpg)
![](https://raw.githubusercontent.com/TrDA-hab/Projects/master/HomeWeatherStation/20200529_004204.jpg)
![](https://raw.githubusercontent.com/TrDA-hab/Projects/master/HomeWeatherStation/WHM-1.jpg)
![](https://raw.githubusercontent.com/TrDA-hab/Projects/master/HomeWeatherStation/20200814_145417.jpg)
![](https://raw.githubusercontent.com/TrDA-hab/Projects/master/HomeWeatherStation/20200908_203218.jpg)
![](https://raw.githubusercontent.com/TrDA-hab/Projects/master/HomeWeatherStation/Graph-1.jpg)

## My PCB (open source hardware). For non-commercial use only!:
![5](https://raw.githubusercontent.com/TrDA-hab/Projects/master/HomeWeatherStation/Schematic_WHM.png)
 - [easyeda_source](https://github.com/TrDA-hab/Projects/tree/master/HomeWeatherStation/easyeda_source)
 - [gerber files](https://github.com/TrDA-hab/Projects/tree/master/HomeWeatherStation/gerber)

**Best regards   
TrDA**

**Comment by sfromis** [comment]
Nice. Was looking for where you keep the BME680 away from the radiator, and where the BH1750 gets to peek out? For low light resolution I'd probably not have used the BH1750 without decimals on the output.
